### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by mumumu77** [moon-wh/MADE#1]
Geat work, thank you for sharing paper and code.
I have following questions.
Q: MADE needs to test data with clothes_ids, right?
    How to evaluate test data without clothes_ids? ex. clothes_ids = 0)?
     https://github.com/moon-wh/MADE/blob/31b9dccecd06f53be931bb7545fbe7cf761fdbab/processor/processor.py#L230

**Comment by moon-wh** [comment]
Yes, our task is based on clothing changing datasets, and the model is trained on the basis of clothes_ids encoding as mentioned in the paper.
